decko/lib/decko/engine.rb

```diff
@@ -15,43 +15,54 @@
 
 module Decko
   class Engine < ::Rails::Engine
-    paths.add "app/controllers",  with: "rails/controllers", eager_load: true
-    paths.add "gem-assets",       with: "rails/assets"
-    paths.add "config/routes.rb", with: "rails/engine-routes.rb"
-    paths.add "lib/tasks", with: "#{::Decko.gem_root}/lib/decko/tasks",
-                           glob: "**/*.rake"
-    paths["lib/tasks"] << "#{::Cardio.gem_root}/lib/card/tasks"
-    paths.add "lib/decko/config/initializers",
-              with: File.join(Decko.gem_root, "lib/decko/config/initializers"),
-              glob: "**/*.rb"
 
-    initializer "decko.engine.load_config_initializers",
-                after: :load_config_initializers do
-      paths["lib/decko/config/initializers"].existent.sort.each do |initializer|
-        load(initializer)
+    engine_name = :decko_engine
```

Comment:
I'd have to study more to understand exactly what you've done here, but it sure looks a lot cleaner!